### ISSUE TITLE
No Icon on Android

### ISSUE THREAD

**Issue by Teyro** [kaie/otp-authenticator-android#1]
Hi,

I dont know if it is the wright Section but i dont get a Icon in my Android Device it is only a "!" with a clock under it....

Did you have no Icon? I can create one!

Best wishes from Hamburg, Germany

Christian

**Comment by kaie** [comment]
It should have an icon. Maybe you have a black background and cannot see
the black icon?

**Comment by Teyro** [comment]
Yes! I use the Nova Launcher und the Android Default Launcher and theese 
to one have black Backgrounds... Cloud you add a bright Version?

Best wishes

Christian

Am 27.08.2013 15:56, schrieb Kai Engert:
